ProtoWorkspace/Constants.fs

```diff
@@ -1,100 +1,192 @@
 ﻿module ProtoWorkspace.Constants
-open System
 
+open System
 
-let [<Literal>] FSharpProjectGuidStr = "F2A71F9B-5D33-465A-A702-920D77279786"
+[<Literal>]
+let FSharpProjectGuidStr = "F2A71F9B-5D33-465A-A702-920D77279786"
 
 let FSharpProjectGuid = Guid FSharpProjectGuidStr
 
-
 // Common Constants
-let [<Literal>] Name        = "Name"
-let [<Literal>] None        = "None"
-let [<Literal>] Reference   = "Reference"
+[<Literal>]
+let Name = "Name"
```

Comment:
another annoying auto format, it wastes space, I'd rather blocks of constants be compact